packages/studioCMSBlog/src/pages/rss.xml.ts

```diff
@@ -0,0 +1,31 @@
+import rss from '@astrojs/rss';
```

Comment:
How is `@astrojs/rss` added to a user's project? Do user's have to install it manually?

This integration could be added using the [`integrations` option from ATP](https://astro-theme-provider.netlify.app/reference/author/#integrations)

Reply:
the RSS thing is not even a real integration...?  I don't add it to any integration array ever... i always just add the page Route and Dependency.

Reply:
O ya 🤦‍♂️ I forgot the package is just a helper for generating RSS, not an integration itself